x/ibc/07-tendermint/update.go

```diff
@@ -22,8 +23,8 @@ import (
 // Tendermint client validity checking uses the bisection algorithm described
 // in the [Tendermint spec](https://github.com/tendermint/spec/blob/master/spec/consensus/light-client.md).
 func CheckValidityAndUpdateState(
```

Comment:
` header height is lower than the latest client height` 

this godoc needs to be updated, this is no longer true